.github/workflows/docker.yaml

```diff
@@ -95,13 +97,24 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Log into registry ghcr.io
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a # pin@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
```

Comment:
It uses the workflow token so it won't get the users permission but will show them as an uploader. That way you know how merged it in instead of the ominous github bot.

Reply:
Sorry, don't get this, I think currently only SchrodingersGat merges things, so this will in all case only a user who has access to the registry, but how does this work Wort the repo token? Is this authorized to login in somebody else's account? 

But now lets say if this would be another account who has no write access to the registry, would that cause an issue? (Or is this no question as nobody else triggers this workflow?)

Reply:
The user is determined by the token, the username is just shown.

Reply:
Oh, so this could be any string. Ok learned something new. Thank you